### PR TITLE
EZP-31951: Redesign modals

### PR DESCRIPTION
**Pull request by lucasOsti**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-31951
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/themes/admin/content/modal/draft_conflict.html.twig [hunk 1]
```diff
@@ -21,21 +21,18 @@
             </div>
             <div class="modal-body">
                 <div class="ez-modal-body__main">
-                    <p class="ez-modal-body__main-content">
                         {% set number = conflicted_drafts|length %}
```
Comment (GrabowskiM) [inline src/bundle/Resources/views/themes/admin/content/modal/draft_conflict.html.twig:24]: I have an impression that after removing this tag you have bad indents in next lines

File: src/bundle/Resources/views/themes/admin/ui/component/custom_dropdown.html.twig [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<div class="ez-custom-dropdown__wrapper {% if class is defined %}{{ class }}{%endif%}" {% if source_selector is defined %}data-source-selector="{{ source_selector }}"{%endif%}>
```
Comment (GrabowskiM) [inline src/bundle/Resources/views/themes/admin/ui/component/custom_dropdown.html.twig:1]: couldn't it be just `class|default('')` insted of `{% if class is defined %}{{ class }}{%endif%}` ?

File: src/bundle/Resources/views/themes/admin/ui/component/custom_dropdown.html.twig [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+                        {{ choice_translation_domain is same as(false) ? choice.label : choice.label|trans({}, choice_translation_domain) }}
+                        <span class="ez-custom-dropdown__remove-selection"></span>
+                    </li>
+                {% endif %}
+            {% endfor %}
+        {% endif %}
+    </ul>
+    <ul class="ez-custom-dropdown__items ez-custom-dropdown__items--hidden">
+        {% if placeholder is defined %}
+            <li data-value="" class="ez-custom-dropdown__item"{% if required and value is empty %} selected="selected"{% endif %}>{{ placeholder != '' ? (translation_domain is same as(false) ? placeholder : placeholder|trans({}, translation_domain)) }}</li>
```
Comment (GrabowskiM) [inline src/bundle/Resources/views/themes/admin/ui/component/custom_dropdown.html.twig:16]: Do not create nested ternary operators

File: src/bundle/ui-dev/src/modules/common/popup/popup.component.js [hunk 4]
```diff
@@ -154,8 +162,13 @@ class Popup extends Component {
             <div {...modalAttrs}>
                 <div className={`modal-dialog c-popup__dialog ${MODAL_SIZE_CLASS[size]}`} role="dialog">
                     <div className="modal-content c-popup__content">
-                        {this.renderHeader()}
-                        <div className="modal-body c-popup__body">{this.props.children}</div>
+                        {noHeader
```
Comment (GrabowskiM) [inline src/bundle/ui-dev/src/modules/common/popup/popup.component.js:165]: couldn't it be single line?

**Review by GrabowskiM — CHANGES_REQUESTED** [review]

**Review by GrabowskiM — APPROVED** [review]

**Comment by dew326** [comment]
Backend is also included here, original PR https://github.com/ezsystems/ezplatform-admin-ui/pull/1534
//cc @adamwojs

**Review by adamwojs — APPROVED** [review]

**Review by dew326 — APPROVED** [review]

**Pull request merged**